word/credential/tool.gpt

```diff
@@ -16,6 +16,8 @@ Tools: ../../oauth2
 			"Fields.Read",
```

Comment:
I think you should replace `Fields.Read` to `File.Read`.  I think `"Fields.Read" are added accidentlly as a typo. The scope doesn't exist

Reply:
Yes, this was my fault.

Reply:
I guess they could be removed as we have ReadWrite below, but I'll leave them here as the Azure permissions played tricks on me more than once :D